olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -270,7 +270,7 @@ spec:
                     - name: OPERATOR_NAME
                       value: "knative-serving-openshift"
                     - name: MIN_OPENSHIFT_VERSION
-                      value: "4.3.0"
+                      value: "4.3.0-0"
```

Comment:
Are you sure this will work on normal installations (not nightly builds) ?

Reply:
The issue is a semver check for a version that is bigger than or equal to what we define here. By semver's spec, a prerelease is smaller than a "proper" release. So this should just be widening the allowed range. Going to add tests though!